src/nfts/NftsInfoCarousel.tsx

```diff
@@ -151,6 +151,10 @@ export default function NftsInfoCarousel({ route }: Props) {
     [NetworkId['arbitrum-sepolia']]: t('viewOnArbiscan'),
     [NetworkId['op-mainnet']]: t('viewOnOPMainnetExplorer'),
     [NetworkId['op-sepolia']]: t('viewOnOPSepoliaExplorer'),
+    [NetworkId['polygon-mainnet']]: t('viewOnPolygonScan'),
+    [NetworkId['polygon-mumbai']]: t('viewOnPolygonScan'),
```

Comment:
can you call this `polygon-pos-mumbai` and `polygon-pos-mainnet`?  That's what we've used elsewhere and it helps differentiate between the different polygon networks (there's a zkEVM-based network from polygon too).